src/libCrypto/EthCrypto.cpp

```diff
@@ -324,7 +324,7 @@ bytes GetOriginalHash(TransactionCoreInfo const& info, uint64_t chainId) {
   rlpStreamRecreated << info.gasLimit;
   rlpStreamRecreated << info.toAddr;
   rlpStreamRecreated << info.amount;
-  rlpStreamRecreated << info.data;
+  rlpStreamRecreated << FromEVM(info.code);
```

Comment:
when building up the rlp stream as it was created on the client side, we need to strip off the 'EVM' prefix otherwise it thinks the signature is incorrect